pkg/ctl/enable/all.go

```diff
@@ -0,0 +1,143 @@
+package enable
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kris-nova/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/git"
+	"github.com/weaveworks/eksctl/pkg/gitops"
+	"github.com/weaveworks/eksctl/pkg/gitops/fileprocessor"
+	"github.com/weaveworks/eksctl/pkg/gitops/flux"
+	"github.com/weaveworks/eksctl/pkg/gitops/profile"
+)
+
+// Options groups input for the "create cluster" command.
+type Options struct {
+	fluxOptions    flux.InstallOpts
+	profileOptions profile.Options
+}
+
+func (opts Options) validate() error {
+	if err := opts.profileOptions.Validate(); err != nil {
+		return err
+	}
+	return cmdutils.ValidateGitOptions(&opts.fluxOptions.GitOptions)
+}
+
+// AllCmd combines the functionality of
+// - eksctl enable repo
+// - eksctl enable profile
+// TODO: eventually have eksctl create cluster run something equivalent to the
+// below, once ClusterConfig supports GitOps settings.
+func AllCmd(cmd *cmdutils.Cmd) {
```

Comment:
I think I would remove this command for now since it will go away. Or what did we decide to do about this?

Reply:
I didn't really decide anything yet, as mentioned here: https://github.com/weaveworks/eksctl/pull/1509#discussion_r341517497
I initially went for 1. but as I was then re-writing `enableProfile` to, well, _just_ enable profiles, I realised it may mean we'd have to trash `gitops.Applier` as well, and I wasn't sure yet about this.

Reply:
I removed it in the end, in a dedicated commit, which I will not squash when merging, so that it can easily be reverted should this be required.